src/helpers/metadataUtils.ts

```diff
@@ -33,8 +33,8 @@ export function generateMetadataParams(ecCurve: ec, serverTimeOffset: number, me
   };
   const sig = key.sign(keccak256(Buffer.from(stringify(setData), "utf8")).slice(2));
   return {
-    pub_key_X: key.getPublic().getX().toString("hex", 64),
-    pub_key_Y: key.getPublic().getY().toString("hex", 64),
+    pub_key_X: key.getPublic().getX().toString("hex"), // DO NOT PAD THIS. BACKEND DOESN'T
```

Comment:
we should update backend isn't it? Coz there might still be 0's in front which can get lost on the network.

Reply:
can't update backend due to legacy reasons.
0's don't get lost because we always transmit strings across network